src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesStatusService.java

```diff
@@ -111,7 +111,7 @@ public void reportStats(int totalObjects, Verbosity verbosity) {
                 showField("Potential Matches", cntPotential);
             }
         } catch (IOException e) {
-            outputLogger.info("Failed to load mappings: {}", e.getMessage());
+            outputLogger.info("Failed to load mappings: {}", e);
```

Comment:
Might make sense to add in a regular logger line here and just send the message to the outputLogger